### PR TITLE
Replace 1/0 with TRUE/FALSE in function help

### PR DESCRIPTION
**Pull request by elpaso**
Homogenize function help when returning booleans

### CODE REVIEW

**Comment by elpaso** [comment]
CC @nyalldawson

File: resources/function_help/json/clamp [hunk 1]
```diff
@@ -19,12 +19,12 @@
...
     "expression": "clamp(1,0,10)",
-    "returns": "1",
-    "note": "<i>input</i> is less than minimum value of 1, so function returns 1"
+    "returns": "TRUE",
+    "note": "<i>input</i> is less than minimum value of 1, so function Returns TRUE"
   }, {
     "expression": "clamp(1,11,10)",
     "returns": "10",
-    "note": "<i>input</i> is greater than maximum value of 10, so function returns 10"
+    "note": "<i>input</i> is greater than maximum value of 10, so function Returns TRUE0"
```
Comment (nyalldawson) [inline resources/function_help/json/clamp:27]: Something wrong here

Comment (elpaso) [reply resources/function_help/json/clamp]: Fixed and checked others.

File: resources/function_help/json/array_find [hunk 2]
```diff
@@ -12,10 +12,10 @@
   }],
   "examples": [{
     "expression": "array_find(array('a', 'b', 'c'), 'b')",
-    "returns": "1"
+    "returns": "TRUE"
```
Comment (DelazJ) [inline resources/function_help/json/array_find:15]:
```suggestion
    "returns": "1"
```

File: resources/function_help/json/array_find [hunk 3]
```diff
@@ -12,10 +12,10 @@
...
   }, {
     "expression": "array_find(array('a', 'b', 'c', 'b'), 'b')",
-    "returns": "1"
+    "returns": "TRUE"
```
Comment (DelazJ) [inline resources/function_help/json/array_find:18]:
```suggestion
    "returns": "1"
```

File: resources/function_help/json/array_intersect [hunk 4]
```diff
@@ -2,7 +2,7 @@
   "name": "array_intersect",
   "type": "function",
   "groups": ["Arrays"],
-  "description": "Returns true if at least one element of array1 exists in array2.",
+  "description": "Returns TRUE if at least one element of array1 exists in array2.",
```
Comment (DelazJ) [inline resources/function_help/json/array_intersect:5]: The example below should output uppercase TRUE for consistency

File: resources/function_help/json/bounds_height [hunk 5]
```diff
@@ -12,7 +12,7 @@
     "returns": "height of bounding box of the current feature's geometry"
   }, {
     "expression": "bounds_height(geom_from_wkt('Polygon((1 1, 0 0, -1 1, 1 1))'))",
-    "returns": "1"
+    "returns": "TRUE"
```
Comment (DelazJ) [inline resources/function_help/json/bounds_height:15]:
```suggestion
    "returns": "1"
```

File: resources/function_help/json/clamp [hunk 6]
```diff
@@ -20,11 +20,11 @@
   }, {
     "expression": "clamp(1,0,10)",
     "returns": "1",
-    "note": "<i>input</i> is less than minimum value of 1, so function returns 1"
+    "note": "<i>input</i> is less than minimum value of 1, so function Returns 1"
```
Comment (DelazJ) [inline resources/function_help/json/clamp:23]:
```suggestion
    "note": "<i>input</i> is less than minimum value of 1, so function returns 1"
```

File: resources/function_help/json/clamp [hunk 7]
```diff
@@ -20,11 +20,11 @@
...
   }, {
     "expression": "clamp(1,11,10)",
     "returns": "10",
-    "note": "<i>input</i> is greater than maximum value of 10, so function returns 10"
+    "note": "<i>input</i> is greater than maximum value of 10, so function Returns 10"
```
Comment (DelazJ) [inline resources/function_help/json/clamp:27]:
```suggestion
    "note": "<i>input</i> is greater than maximum value of 10, so function returns 10"
```

File: resources/function_help/json/contains [hunk 8]
```diff
@@ -2,7 +2,7 @@
   "name": "contains",
   "type": "function",
   "groups": ["GeometryGroup"],
-  "description": "Tests whether a geometry contains another. Returns true if and only if no points of geometry2 lie in the exterior of geometry1, and at least one point of the interior of geometry2 lies in the interior of geometry1.",
+  "description": "Tests whether a geometry contains another. Returns TRUE if and only if no points of geometry2 lie in the exterior of geometry1, and at least one point of the interior of geometry2 lies in the interior of geometry1.",
```
Comment (DelazJ) [inline resources/function_help/json/contains:5]: Update true/false examples returns to uppercase. Affects many functions...

File: resources/function_help/json/m_max [hunk 9]
```diff
@@ -9,7 +9,7 @@
   }],
   "examples": [{
     "expression": "m_max( make_point_m( 0,0,1 ) )",
-    "returns": "1"
+    "returns": "TRUE"
```
Comment (DelazJ) [inline resources/function_help/json/m_max:12]:
```suggestion
    "returns": "1"
```

File: resources/function_help/json/op_minus [hunk 10]
```diff
@@ -12,7 +12,7 @@
   }],
   "examples": [{
     "expression": "5 - 4",
-    "returns": "1"
+    "returns": "TRUE"
```
Comment (DelazJ) [inline resources/function_help/json/op_minus:15]:
```suggestion
    "returns": "1"
```

File: resources/function_help/json/radians [hunk 11]
```diff
@@ -12,7 +12,7 @@
     "returns": "3.14159"
   }, {
     "expression": "radians(57.2958)",
-    "returns": "1"
+    "returns": "TRUE"
```
Comment (DelazJ) [inline resources/function_help/json/radians:15]:
```suggestion
    "returns": "1"
```

File: resources/function_help/json/z_min [hunk 12]
```diff
@@ -9,10 +9,10 @@
   }],
   "examples": [{
     "expression": "z_min( geom_from_wkt( 'POINT ( 0 0 1 )' ) )",
-    "returns": "1"
+    "returns": "TRUE"
```
Comment (DelazJ) [inline resources/function_help/json/z_min:12]:
```suggestion
    "returns": "1"
```

File: resources/function_help/json/z_min [hunk 13]
```diff
@@ -9,10 +9,10 @@
...
   }, {
     "expression": "z_min( geom_from_wkt( 'MULTIPOINT ( 0 0 1 , 1 1 3 )' ) )",
-    "returns": "1"
+    "returns": "TRUE"
```
Comment (DelazJ) [inline resources/function_help/json/z_min:15]:
```suggestion
    "returns": "1"
```

**Review by DelazJ — COMMENTED** [review]
Happens that not all `returns 1` are meant to be boolean.
It could also be nice to update the existing returns true to TRUE for a single spelling.
My question now: is this only for the help text? Should the expression preview (as well as its output in field or whatever) also not render TRUE instead of 1/0?

File: resources/function_help/json/intersects [hunk 14]
```diff
@@ -15,7 +15,7 @@
     "returns": "true"
```
Comment (elpaso) [inline resources/function_help/json/intersects:15]:
```suggestion
    "returns": "TRUE"
```

File: resources/function_help/json/contains [hunk 15]
```diff
@@ -15,7 +15,7 @@
     "returns": "true"
```
Comment (elpaso) [inline resources/function_help/json/contains:15]:
```suggestion
    "returns": "TRUE"
```

File: resources/function_help/json/touches [hunk 16]
```diff
@@ -15,7 +15,7 @@
     "returns": "true"
```
Comment (elpaso) [inline resources/function_help/json/touches:15]:
```suggestion
    "returns": "TRUE"
```

File: resources/function_help/json/within [hunk 17]
```diff
@@ -15,7 +15,7 @@
     "returns": "true"
```
Comment (elpaso) [inline resources/function_help/json/within:15]:
```suggestion
    "returns": "TRUE"
```

**Pull request merged**